src/server/orchestrator.ts

```diff
@@ -8,32 +6,14 @@ import { TurnResultData } from "../shared/events";
 import { PlayerInfo, StoreClient } from "./store/store";
 import { Word } from "../shared/word";
 import Player from "../shared/player";
-
-const isStaging = DAILY_STAGING === "true";
-
-let dailyAPIDomain = "daily.co";
-if (isStaging) {
-  dailyAPIDomain = "staging.daily.co";
-}
-
-const dailyAPIURL = `https://api.${dailyAPIDomain}/v1`;
-
-// The data we'll expect to get from Daily on room creation.
-// Daily actually returns much more data, but these are the only
-// properties we'll be using.
-interface ICreatedDailyRoomData {
-  id: string;
-  name: string;
-  url: string;
-}
+import { createRoom, tokenIsValid } from "./daily";
```

Comment:
Here we're moving Daily operations out of the Orchestrator into their own module for easier parsing (and writing)